src/components/Tree/Node.tsx

```diff
@@ -19,32 +18,44 @@ export type TreeNodeProps = {
 type NodeProps = {
     node: TreeNodeProps;
     strong?: boolean;
-    activeNodeId?: string;
-    onClick: (id: string) => void;
+    activeNodeId?: string | null;
+    parentIds?: string[];
+    onClick: (id: string | null) => void;
 };
 
 export default function TreeNode({
     node: { id, value, name, label, icon, nodes },
     strong = false,
-    activeNodeId,
+    activeNodeId = null,
     onClick,
+    parentIds = [],
 }: NodeProps): ReactElement<NodeProps> {
     const [showNodes, setShowNodes] = useState(false);
+    const selected = id === activeNodeId;
 
     return (
         <li data-test-id="node">
             <a
+                data-test-id="node-link"
                 className={merge([
-                    css.nodeLink,
-                    id === activeNodeId && css.nodeLinkSelected,
-                    strong && css.strong,
-                    value && css.selectableNodeLink,
+                    "flex items-center justify-center py-1 px-2 rounded text-black cursor-pointer no-underline",
+                    strong && "font-bold",
+                    value && !selected && "hover:bg-black-5",
+                    selected && "bg-violet-60 text-white",
```

Comment:
Please rewrite this to `selected ? "bg-violet-60 text-white" : "text-black"` otherwise it possible does not do what you intent it to do... (since the classes do not overwrite each other, if you know what i mean)

Reply:
👍 11a17cf